engines/registry/app/models/access_scope.rb

```diff
@@ -88,19 +88,19 @@ def allowed_paths(system = nil)
     allowed_products = (active_products & access_policies_yml.keys)
     allowed_glob_paths = access_policies_yml.values_at(*allowed_products).flatten
 
-    if system && system.hybrid?
-      allowed_product_class = allowed_products.detect { |s| s.downcase.include?('ltss') }
-      if allowed_product_class.present?
-        activation_state = SccProxy.scc_check_subscription_expiration(
-          {}, system.login, system.system_token, Rails.logger, system.proxy_byos_mode, allowed_product_class
+    if system && system.hybrid? && allowed_products.any? { |s| !Product.find_by(product_class: s).free? }.present?
```

Comment:
We'll need a comment here what this condition is doing. I guess it's something like `# if the system is hybrid, check if it's subscription is still valid for accessing images`
Why is this only checking for hybrid systems and not BYOS?